src/server/rest.cpp

```diff
@@ -79,6 +95,11 @@ namespace ouroboros
 
 		std::string result;
 		result.assign(match[0].ptr, match[0].len);
+		
+		if(result != "server"){
+			result.insert(0, "server-");
+		}
+
```

Comment:
Ya, we need some consensus on this. With this current implementation, the user can return the entire root group's JSON using: localhost:8080/groups/server. The problem exists when the user creates a group called "server" directly in the root group. Our system would return the root group JSON instead of the user created groups JSON. We will need a reserved word here specifically for accessing the root group.